README.md

```diff
@@ -28,12 +28,13 @@ make check
 ## Usage
 
 ```
-build/kvm-host -k bzImage [-i initrd]
+build/kvm-host -k bzImage [-i initrd] [-d disk-image]
 ```
 
 `bzImage` is the path to linux kernel bzImage. The bzImage file is in a specific format,
 containing concatenated `bootsect.o + setup.o + misc.o + piggy.o`. `initrd` is the path to
 initial RAM disk image, which is an optional argument.
+`disk-image` is the path to ext4 disk image. The disk image is as a block device with virtio-blk in Guest Linux.
```

Comment:
The disk image is not restricted to ext4, right? If so, I think we have to fix the description here.

By the way, can we provide either scripts or more information to build the disk image?

Reply:
> The disk image is not restricted to ext4, right?
Yes, but we only enable the ext4 support on `linux.config`.